generators/contract_report.py

```
@@ -1,334 +1,447 @@
+        # 4. 계약 조건
+        contract_parts.append("=== 제3조 임대차 기간 및 임료 ===")
+        
+        # 날짜 파싱
+        try:
+            start_date = datetime.strptime(basic_info.contract_start_date, '%Y-%m-%d')
+            end_date = datetime.strptime(basic_info.contract_end_date, '%Y-%m-%d')
+            period_days = (end_date - start_date).days + 1
+            period_years = period_days / 365
+            
+            contract_parts.append(f"계약기간: {start_date.strftime('%Y년 %m월 %d일')} ~ {end_date.strftime('%Y년 %m월 %d일')} (총 {period_days}일, 약 {period_years:.1f}년)")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)